aten/src/ATen/native/native_functions.yaml

```diff
@@ -131,6 +131,40 @@
     CPU: _abs_out_cpu
     CUDA: _abs_out_cuda
 
+- func: real(Tensor self) -> Tensor
+  variants: function, method
+  named_guard: False
+
+- func: real_(Tensor(a!) self) -> Tensor(a!)
```

Comment:
`real_` changes the dtype of a tensor, so I'm not sure how you can actually do it inplace...

Reply:
These should absolutely NOT be inplace. Thanks for catching this.